.storybook/preview.js

```diff
@@ -1,4 +1,10 @@
+import React from 'react'
+import { addDecorator } from '@storybook/react'
+import { MemoryRouter } from 'react-router'
 
+addDecorator((story) => (
+  <MemoryRouter initialEntries={['/']}>{story()}</MemoryRouter>
```

Comment:
App.tsx is using `BrowserRouter`... why do we want to use `MemoryReouter` in storybook please?

Reply:
I am using `NavLink` inside `homePage` component and while running storybook I was facing error similar to this issue 
https://github.com/storybookjs/storybook/issues/769#issuecomment-294082103
https://stackoverflow.com/questions/58909666/storybook-w-react-router-you-should-not-use-link-outside-router

Though I just verified I can also use BrowserRouter instead of MemoryRouter. I remember storybook won't recognize any navigation actions without a decorator. Please suggest if there's any other way to eliminate this problem.

Reply:
oh, right, there is not point for navigation inside storybook,, so it doesn't matter which router provider we use inside storybook to make the warnings go away :) ok ok